packages/contentful--app-scripts/lib/utils.js

```diff
@@ -125,8 +127,64 @@ function getActionsManifest() {
   } catch {
     console.log(
       `${chalk.red('Error:')} Invalid JSON in manifest file at ${chalk.bold(
-        DEFAULT_MANIFEST_PATH
-      )}.`
+        DEFAULT_MANIFEST_PATH,
+      )}.`,
+    );
+    // eslint-disable-next-line no-process-exit
+    process.exit(1);
+  }
+}
+
+function getDeliveryFunctionsManifest() {
```

Comment:
Could we somehow merge `getActionsManifest` and `getDeliveryFunctionsManifest` methods? They are both parsing the same file. When reading this code first, I was confused why we now have two manifests.